Mathlib/MeasureTheory/Measure/TightNormed.lean

```diff
@@ -0,0 +1,70 @@
+/-
+Copyright (c) 2025 Rémy Degenne. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Rémy Degenne
+-/
+import Mathlib.Analysis.Normed.Module.FiniteDimension
+import Mathlib.Analysis.Normed.Order.Lattice
+import Mathlib.MeasureTheory.Measure.Tight
+import Mathlib.Order.CompletePartialOrder
+
+/-!
+# Tight sets of measures in normed spaces
+
+Criteria for tightness of sets of measures in normed and inner product spaces.
+
+## Main statements
+
+* `isTightMeasureSet_iff_tendsto_measure_norm_gt`: in a finite dimensional real normed space,
+  a set of measures `S` is tight if and only if the function `r ↦ ⨆ μ ∈ S, μ {x | r < ‖x‖}`
+  tends to `0` at infinity.
+
+-/
+
+open Filter
+
+open scoped Topology
+
+namespace MeasureTheory
+
+variable {E : Type*} [NormedAddCommGroup E] {mE : MeasurableSpace E} {S : Set (Measure E)}
+
+section NormedSpace
+
+lemma tendsto_measure_norm_gt_of_isTightMeasureSet (hS : IsTightMeasureSet S) :
+    Tendsto (fun r : ℝ ↦ ⨆ μ ∈ S, μ {x | r < ‖x‖}) atTop (𝓝 0) := by
+  suffices Tendsto ((⨆ μ ∈ S, μ) ∘ (fun r ↦ {x | r < ‖x‖})) atTop (𝓝 0) by
+    convert this with r
+    simp
+  refine hS.comp <| .mono_right ?_ <| monotone_smallSets Metric.cobounded_le_cocompact
+  refine HasAntitoneBasis.tendsto_smallSets ?_
+  exact ⟨Filter.atTop_basis_Ioi.cobounded_of_norm, fun _ _ hr x ↦ hr.trans_lt⟩
+
+section FiniteDimensional
+
+variable [NormedSpace ℝ E] [FiniteDimensional ℝ E]
```

Comment:
Instead of assuming this, could you assume `ProperSpace E`? The proofs are exactly the same, but it is more general (and has fewer algebraic requirements)